tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -71,9 +72,13 @@ def test_fsx_lustre_configuration_options(
     imported_file_chunk_size,
 ):
     mount_dir = "/fsx_mount_dir"
-    bucket_name = s3_bucket_factory()
-    bucket = boto3.resource("s3", region_name=region).Bucket(bucket_name)
-    bucket.upload_file(str(test_datadir / "s3_test_file"), "s3_test_file")
+    bucket_name = None
+    if deployment_type != "PERSISTENT_2":
+        # Association to S3 is currently not supported with Persistent 2 because it is not supported by CloudFormation.
+        # FSx is working on supporting it through CloudFormation
+        bucket_name = s3_bucket_factory()
```

Comment:
We need to cover this by the validators, to not permit having ImportedFileChunkSize, Export/ImportPath when using PERSISTENT_2

Reply:
As discussed, we can keep it and wait for the FSx support